File: jput-core/src/main/java/cz/encircled/jput/io/TrendResultWriter.java
package cz.encircled.jput.io;

import cz.encircled.jput.model.PerformanceTestRun;

/**
 * @author Vlad on 21-May-17.
 */
public interface TrendResultWriter {

    void appendTrendResult(PerformanceTestRun run);

    void flush();

}

File: jput-core/src/main/java/cz/encircled/jput/io/TrendResultWriter.kt
package cz.encircled.jput.io

import cz.encircled.jput.model.PerformanceTestExecution

/**
 * @author Vlad on 21-May-17.
 */
interface TrendResultWriter {

    fun appendTrendResult(execution: PerformanceTestExecution)

    fun flush()

}
